app/main.py

```diff
@@ -806,6 +807,19 @@ def get_owner_email(owner_id):
     else:
         return jsonify({"email": res[0].email})
 
+# Get information of the latest body scaffold for species.
+# This endpoint returns the metadata file path, bucket,
+# dataset id and version which can be used to construct the url
+@app.route("/get_body_scaffold_info/<species>", methods=["GET"])
+def get_body_scaffold_info(species):
+    # Filter to find user based on provided int id
+    id = get_body_scaffold_dataset_id(species)
+    if id:
+        result = process_get_first_scaffold_info(dataset_search(create_pennsieve_identifier_query(id)))
+        if result:
+            return result
+
+    return abort(400, description=f"Whole body info not found for {species}")
```

Comment:
would 404 be better here? or a json response with 'no scaffold for this species'

Can send a valid request with:
http://localhost:5000/get_body_scaffold_info/dog

and we get back this:
![image](https://user-images.githubusercontent.com/37255664/228695073-868d2682-8026-410c-99b8-c5d8b9b1c15c.png)

Reply:
Yes, 404 is better.